keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/README.md

```diff
@@ -831,19 +774,10 @@ details). For now, we leave it here.
 N/A
 
 ###### What are other known failure modes?
-N/A
-<!--
-For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
--->
+The failure modes is similar to existing options. It changes the way how cpu manager allocate CPUs.
```

Comment:
I'l defer this part to PRR reviewers. I think we may want to mention that this option should not make the cpu allocation less likely (does it?)

Reply:
Actually, in our downstream version, we did have an issue that reserved witholding logic has some issue and leads to few cpus available. This has been fixed and now it won't have such issues.